### ISSUE TITLE
Request: randomly blame someone else.

### ISSUE THREAD

**Issue by joshmanders** [jayphelps/git-blame-someone-else#13]
This is great, thanks for making it. Can we not supply someone to blame and have it pick a random author to blame instead of me?

Thanks.

**Comment by gdi2290** [comment]
![](http://blogs-images.forbes.com/roberthof/files/2011/09/google-1-plus-button.png)

**Comment by kelunik** [comment]
:+1:

**Comment by kamranahmedse** [comment]
Yeah, will be suitable to `blame-someone-else` without feeling any remorse of targeting someone specifically *fate my friend

**Comment by garethellis0** [comment]
Done, see pull request
